src/main/cpp/subsystems/Wrist.cpp

```diff
@@ -0,0 +1,56 @@
+#include "subsystems/Wrist.h"
+#include "Constants.h"
+
+#include "frc2/command/FunctionalCommand.h"
+
+Wrist::Wrist() {
+    MotorUtils::Motor::LogValues logValues {true, true, true};
+    MotorUtils::Motor wristMotor{&m_motor, CONSTANTS::WRIST::PidValue, logValues};
+    AddPID(wristMotor);
+    SetPID();
+
+    m_motor.SetPosition(0_tr);
```

Comment:
`0_tr` is not explicitly a safe value. The stowed/default value should be in Constants